### PR TITLE
Adds descriptions for named keys using bulk insert

### LINKED ISSUE #337: Post multiple values in JSON format
Hello,
is it possible to send multiples values in JSON format, with a timestamp ?

Perhaps something like that :
http://emoncms.org/input/post.json?json={{power:200,offset:-10},{temperature:25,offset: -5}}&time=1433504598

### PR DESCRIPTION
**Pull request by fredrike**
Closes #337 together with #615.

### CODE REVIEW

File: Modules/input/Views/input_api.php [hunk 1]
```diff
@@ -52,6 +52,7 @@
 <tr><td><?php echo _('Time offset format (-6 is -16 seconds to now):'); ?></td><td><a href="<?php echo $path; ?>input/bulk.json?data=[[-10,16,1137],[-8,17,1437,3164],[-6,19,1412,3077]]&offset=-10"><?php echo $path; ?>input/bulk.json?data=[[-10,16,1137],[-8,17,1437,3164],[-6,19,1412,3077]]<b>&offset=-10</b></a></td></tr>
 <tr><td><?php echo _('Sentat format: (useful for sending as positive increasing time index)'); ?></td><td><a href="<?php echo $path; ?>input/bulk.json?data=[[520,16,1137],[530,17,1437,3164],[535,19,1412,3077]]&sentat=543"><?php echo $path; ?>input/bulk.json?data=[[520,16,1137],[530,17,1437,3164],[535,19,1412,3077]]<b>&sentat=543</b></b></a></td></tr>
 <tr><td><?php echo _('Absolute time format (-6 is 1387730121 seconds since 1970-01-01 00:00:00 UTC))'); ?></td><td><a href="<?php echo $path; ?>input/bulk.json?data=[[-10,16,1137],[-8,17,1437,3164],[-6,19,1412,3077]]&time=<?php echo time(); ?>"><?php echo $path; ?>input/bulk.json?data=[[-10,16,1137],[-8,17,1437,3164],[-6,19,1412,3077]]<b>&time=<?php echo time(); ?></b></a></td></tr>
```
Comment (fredrike) [inline Modules/input/Views/input_api.php]: There is an extra ')' after UTC, but that is not part of this commit.

**Comment by TrystanLea** [comment]
Great thanks!
